tinygrad/runtime/ops_python.py

```diff
@@ -82,18 +85,20 @@ def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tup
           loop_ends[idp[0]] = i
           i = idp[0]
           continue
-        elif uop is UOps.BARRIER:
+        # TODO: implement UOps.IF and UOps.ENDIF
+        elif uop in (UOps.BARRIER, UOps.IF, UOps.ENDIF):
```

Comment:
I had to skip `UOps.IF` and `UOps.ENDIF` temporarily similar to `UOps.BARRIER` so I could focus on another fix, though to my surprise this didn't cause any test to fail. So I pushed this for now while I investigate it further.

Reply:
Yea, it's actually okay to not have IF/ENDIF, it's an optional performance improvement for GPU, not a requirement.

BARRIER isn't needed because it's always in sync in this interpreter

Reply:
I see. Thanks for clarifying this.